DigitalLearningSolutions.Web/Views/Support/Faqs/Faqs.cshtml

```diff
@@ -23,20 +27,25 @@
     <input type="hidden" id="select-sort-by" value="@Model.SortBy" data-sort-by-multiple="true" />
     <input type="hidden" id="select-sort-direction" value="@Model.SortDirection" />
 
-    @if (Model.NoDataFound) {
+    @if (Model.NoDataFound)
+    {
       <p class="nhsuk-u-margin-top-4" role="alert">
         <b>No FAQs found.</b>
       </p>
-    } else {
+    }
+    else
+    {
       <partial name="SearchablePage/_SearchResultsCount" model="Model" />
 
       <div id="searchable-elements">
-        @foreach (var faq in Model.Faqs) {
+        @foreach (var faq in Model.Faqs)
+        {
           <partial name="_SearchableFaqCard" model="faq" />
         }
       </div>
     }
-    @if (Model.TotalPages > 1) {
+    @if (Model.TotalPages > 1)
+    {

```

Comment:
This auto-format has made this code inconsistent with code style in other cshtml files. Have you ever worked out how to modify VS's settings for cshtml style?

Reply:
I never dug any deeper. It doesn't happen all that often. But thanks for the prompt I'm going to look a little deeper.
Also I've sorted the formatting.